Nav bar

sehilyi opened the pull request:
Added a simple navigation bar on the top with links for GitHub and documentations.

Towards #235

keller-mark commented on src/demo/App.scss:
```diff
@@ -1,24 +1,50 @@
+            padding-left: 15px;
+            padding-right: 15px;
+            .viewconf-options {
+                margin-left: 10px;
```
```suggestion
                margin-left: 10px;
                font-size: 14px;
```

keller-mark commented on src/demo/App.scss:
```diff
@@ -1,24 +1,50 @@
     text-align: center;
+    .header-container{
+        font-family: 'Lato', sans-serif;
+        font-weight: bold;
```
```suggestion
        font-weight: normal;
```

keller-mark commented:
Can we let the `.container` element scroll on its own so that the nav bar does not move with it? I imagine that to do this, we will need to give the nav bar and container elements fixed heights

or `.nav-bar{ height: 70px; }` and `.container { height: calc(100% - 70%); }` and also give the `body { height: 100vh; }` and then add `.container { overflow-x: scroll; }`?
![Screen Shot 2020-05-18 at 12 02 19 PM](https://user-images.githubusercontent.com/7525285/82234515-85608880-98ff-11ea-80dd-33adddd19955.png)

sehilyi commented:
Thanks for sharing the screenshot to me, I had noticed the bad alignment of components on the top when the width is too narrow. I've fixed this alignment issue and updated to support horizontal scrolling separately for the nav bar and the main view on the bottom. 

![Screen Shot 2020-05-18 at 3 19 53 PM](https://user-images.githubusercontent.com/9922882/82251406-26107180-991b-11ea-8fe3-694482de933b.png)

There still is an issue w/ scrolling; the wrapper components on the left are still hidden. Since this issue likely to be no problem for our demo for ITCR, let's fix this issue within a separate PR.
